notebooks/amazon_sagemaker_and_chroma_for_qa.ipynb

```diff
@@ -0,0 +1,508 @@
+{
```

Comment:
<p>jfyi, you mention "follow this guide to understand" but there's no link</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>